### PR TITLE
TPROD-281 - Test kernel boot fix for Symfony 5

### PR DESCRIPTION
**Pull request by fedys**
| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [x]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | 
| Related developer documentation PR URL |  
| Issue(s) addressed                     | https://mautic.atlassian.net/browse/TPROD-281 

#### Description:
This PR makes functional tests ready for Symfony 5 upgrade.

#### Steps to test this PR:

1. This PR touches just tests. Just passing CI pipeline is fine. No manual test steps are needed.

<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/11045"></a>

### CODE REVIEW

**Comment by fedys** [comment]
I think we can ignore the decreased code coverage. I touched only tests and didn't remove any assertion/test logic.

**Review by escopecz — APPROVED** [review]
Thanks! 👍

**Comment by escopecz** [comment]
Merging as this is modifying just the test code (no production code) and the tests are passing. I don't think we need a second review here.

**Pull request merged**
